ci/do_ci.sh

```diff
@@ -238,6 +243,8 @@ if [ -n "$CIRCLECI" ]; then
     fi
     echo "Running with ${NUM_CPUS} cpus and BAZEL_BUILD_OPTIONS: ${BAZEL_BUILD_OPTIONS}"
     BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS} --jobs=${NUM_CPUS}"
+else
+    echo "Running with ${NUM_CPUS} cpus and BAZEL_BUILD_OPTIONS: ${BAZEL_BUILD_OPTIONS}"
```

Comment:
How about consolidating this with line 244 and moving it out of the if..fi block?

Reply:
Good point, done in 4a566e9c11f64973319b5fa650049c48c8bc17ff.